src/helpers/walletTypes.ts

```diff
@@ -0,0 +1,14 @@
+import { EthereumWalletType } from '../model/wallet';
+
+const walletTypes: AllWalletTypes = {
```

Comment:
doesn't typescript have enums? can we use those for our constants

Reply:
YES! much better!